inductiva/tasks/task.py

```diff
@@ -178,3 +178,21 @@ def get_execution_time(self) -> float:
                                                 "%Y-%m-%dT%H:%M:%S.%f+00:00")
 
         return (end_time - start_time).total_seconds()
+
+    def get_machine(self) -> str:
```

Comment:
Since you're returning the machine type, should this be named `get_machine_type`?